cmd/server/main.go

```diff
@@ -17,18 +17,24 @@ package main
 import (
 	"os"
 
+	adapter "istio.io/mixer/adapter"
 	"istio.io/mixer/cmd/server/cmd"
 	"istio.io/mixer/cmd/shared"
+	pkgAdapter "istio.io/mixer/pkg/adapter"
 	"istio.io/mixer/pkg/template"
 	generatedTmplRepo "istio.io/mixer/template"
 )
 
-func getSupportedTemplateInfos() template.Repository {
-	return template.NewRepository(generatedTmplRepo.SupportedTmplInfo)
+func getSupportedTemplateInfos() map[string]template.Info {
```

Comment:
`supportTemplates`

Do not put `Get` in the name of a getter method in golang. https://golang.org/doc/effective_go.html#Getters